docs/content/quickstart/parameters.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "QuickStart: Parameters"
+descriptions: Learn how to use a bundle with parameters
+layout: single
+---
+
+Now that you know how to install a bundle, let's look at how to specify parameters to customize how that bundle is installed.
+Bundles can define parameters to allow the end-user to tweak how the bundle is configured.
+A parameter can be a string, integer, boolean or even a json object.
+Some examples of how parameters can be used in a bundle are:
+
+* Log Level: Default the log level for an application to info. At any time a user can upgrade the bundle to change that parameter to a different value.
+* Deployment Region: Let the user specify which region, such as eastus-1, where the application should be deployed.
+* Helm Release Name: A bundle that uses Helm often will define a parameter that allows the user to set the release name for the Helm release.
+
+For optional parameters, bundles set a default value that is used when the user does not specify a value for the parameter.
```

Comment:
To make the parameter optional, you must set a default value for the parameter within the bundle. 

Q: is the parameter optional here? Or really just optional at being specified. Just think this doesn't feel exactly precise.

Reply:
Just realized in the quickstart that the parameters are 

namespace
wordpress-name
wordpress-password 

and both namespace and wordpress-name are optional (required is set to false) and only wordpress-name has a default setting.

Reply:
I'm a bit lost here and I think some of the confusion may being caused by the old/unused parameter field `required`. That isn't supported anymore to specifying if a parameter must be specified or not. If a default is set, then the parameter is optional and does not need to be specified by the user. If a default is not set, then the parameter is required. In a separate PR I will clean up any example porter.yaml files that are still using `required: false` or anything like that.

Does that help clear up this question? Or does the text still need to change?

Reply:
Ahhhhhh! The output is confusing which is why I'm lost. so I think there is more nuance here and I'll need to chat to you more about it because optional feels like the wrong word